lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -1,35 +1,45 @@
 <div class="page-finish row around-xs">
   <div class="col-xs">
-    <div class="box text-center">
-      <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flash Complete!</h3>
-      <p class="soft space-vertical-small" ng-show="finish.settings.get('unmountOnSuccess')">Safely ejected and ready for use</p>
+    <div class="box center">
+      <div class="col-xs-5">
+        <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flash Complete!</h3>
+      </div>
 
-      <div class="row center-xs space-vertical-medium">
-        <div class="col-xs-4 space-medium">
-          <div class="box">
-            <p class="soft button-label">Would you like to flash the same image?</p>
+      <div class="col-xs-4">
+        <button class="button button-primary button-brick" ng-click="finish.restart({ preserveImage: true })">
+          <b>Flash Another</b>
+        </button>
+      </div>
+    </div>
 
-            <button class="button button-primary button-brick" ng-click="finish.restart({ preserveImage: true })">
-              Use <b>same</b> image
-            </button>
-          </div>
+    <div class="box center">
+      <safe-webview src="'https://etcher.io/etcher_banner/'"
+        timeout="2000"
+        callback="finish.setShowWebview"></safe-webview>
```

Comment:
I think I mentioned at some point, but can we change this to a `visible` boolean property that gets set or unset accordingly?